libcxx/test/std/utilities/variant/variant.variant/variant.assign/copy.pass.cpp

```diff
@@ -22,88 +22,108 @@
 #include "test_macros.h"
 
 struct NoCopy {
-  NoCopy(const NoCopy &) = delete;
-  NoCopy &operator=(const NoCopy &) = default;
+  NoCopy(const NoCopy&)            = delete;
```

Comment:
For the future it would be nice to do the formatting is a separate patch/commit that makes it easier to see the real changes.

Reply:
yes this made me confused. I thought we've already clang-formatted everything and I am surprised to see formatting changes

Reply:
We only did the source files. Not the tests and I'm not sure about the benchmarks.

Reply:
ah I see. thanks!